applications/MeshingApplication/python_scripts/mmg_process.py

```diff
@@ -3,6 +3,17 @@
 import KratosMultiphysics as KratosMultiphysics
 import KratosMultiphysics.MeshingApplication as MeshingApplication
 
+try:
+    import KratosMultiphysics.StructuralMechanicsApplication as StructuralMechanicsApplication
```

Comment:
why do you need to import it here?
from what I see you only use it in ExecuteFinalize => why not importing it there?

Reply:
For the SPRProcess